var/spack/repos/builtin/packages/usalign/package.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Usalign(Package):
+    """US-align (Universal Structural alignment) is a unified protocol
+    to compare 3D structures of different macromolecules (proteins,
+    RNAs and DNAs) in different forms (monomers, oligomers and
+    heterocomplexes) for both pairwise and multiple structure
+    alignments."""
+
+    homepage = "https://zhanggroup.org/US-align"
+    url = "https://zhanggroup.org/US-align/bin/module/USalign.cpp"
```

Comment:
This is a highly unusual URL but I appreciate that the other options I noticed on the web site are all zip files of platform-specific binaries.

Reply:
Perhaps it's worth adding a comment to explain why only that file is being downloaded?

Reply:
Yeah, this group doesn't seem to do actual versioning, just distributing the source as a single cpp file that's (seemingly) periodically updated. They do the same with the binaries as well, no version specific files, just the one file. I can add in a note explaining that the source is distributed as a single source file.